cime_config/machines/config_machines.xml

```diff
@@ -1810,6 +1810,9 @@
       <env name="NETCDF_FORTRAN_PATH">$SHELL{dirname $(dirname $(which nf-config))}</env>
       <env name="PNETCDF_PATH">$SHELL{dirname $(dirname $(which pnetcdf_version))}</env>
     </environment_variables>
+    <environment_variables mpilib="openmpi">
+      <env name="OMPI_MCA_sharedfp">^lockedfile</env>
```

Comment:
Is that "^" supposed to be there?

Reply:
Yes, `^` is "not" to prevent lock files.
```
$ ompi_info --param sharedfp all 
            MCA sharedfp: individual (MCA v2.1.0, API v2.0.0, Component
                          v4.1.3)
            MCA sharedfp: lockedfile (MCA v2.1.0, API v2.0.0, Component
                          v4.1.3)
            MCA sharedfp: sm (MCA v2.1.0, API v2.0.0, Component v4.1.3)
```
More info at https://www.open-mpi.org/faq/?category=ompio

Reply:
ok